acdc-ws/conf/application.conf

```diff
@@ -16,12 +16,20 @@
 #mykey = ${JAVA_HOME}
 
 acdc.auth = {
-    enabled = false
-    header-name = "x-api-key"
-    hashed-keys = {
-        user = [ ${?MCE_ENV_X_API_USER1} , ${?MCE_ENV_X_API_USER2} ]
-        admin = [ ${?MCE_ENV_X_API_ADMIN1} , ${?MCE_ENV_X_API_ADMIN2} ]
+    apikey = {
```

Comment:
this would be a breaking change, maybe make all the breaking change when we remove the roles?

Reply:
good catch. I'll include it in the role removal PR.